hooks/package.json

```
@@ -37,11 +41,14 @@
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "jsdom": "^26.1.0",
+    "jsdom": "^26.1.0",
```

Biome (1.9.4)

[error] The key jsdom was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)